### PR TITLE
feat(kudoctl): add get resources command

### PR DESCRIPTION
**Pull request by NilsPonsard**
This PR adds the `get resources` command.

I’m not sure where to put the implementation of the custom trait `HumanReadable` for `GetWorkloadResponse`. By default I put it in the subcommand execution folder.

### CODE REVIEW

File: kudoctl/src/subcommands/get/output.rs [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+    match format {
+        OutputFormat::HumanReadable => output.human_readable(header, 80),
+        OutputFormat::Json => serde_json::to_string(&output).map_err(anyhow::Error::from),
+        OutputFormat::Yaml => serde_yaml::to_string(&output).map_err(anyhow::Error::from),
+    }
+    .context("Error formatting output")
+}
+
+/// Formats the output of a `get` command in the human readable format.
+pub trait HumanReadable {
```
Comment (sameo) [inline kudoctl/src/subcommands/get/output.rs:30]: Can you please consider implementing the [Display](https://doc.rust-lang.org/std/fmt/trait.Display.html) trait for it instead?

Comment (NilsPonsard) [reply kudoctl/src/subcommands/get/output.rs:30]: @sameo I considered the option, but i need to pass the `header` bool argument to it and I don’t know how to do it with the Display trait.

Comment (NilsPonsard) [reply kudoctl/src/subcommands/get/output.rs:30]: Another thing that I wanted with this custom trait is the ability to limit the width of an output (with `max_col`).

Comment (sameo) [reply kudoctl/src/subcommands/get/output.rs:30]: You could carry these options through e.g. the `GetWorkloadResponse` structure and implement `Display` on it.

Comment (NilsPonsard) [reply kudoctl/src/subcommands/get/output.rs:30]: This would cause problems with the `Serialize` trait.

Comment (NilsPonsard) [reply kudoctl/src/subcommands/get/output.rs:30]: Oh this should work, I’ll try that.

Comment (sameo) [reply kudoctl/src/subcommands/get/output.rs]: Looks good. Can you please squash the 2 commits into a single one?

Comment (NilsPonsard) [reply kudoctl/src/subcommands/get/output.rs]: I figured that this method won’t work anyway, in `get/mod.rs` the match statement returns multiple types (one `resources`, one for `resource`, one for `instance`...). I need to find a better way to handle printing the output.

Comment (NilsPonsard) [reply kudoctl/src/subcommands/get/output.rs]: I tried a new way to handle my problem, what do you think ?

**Review by sameo — COMMENTED** [review]
LGTM. Please squash the first 3 commits into 1, and expand the last commit message into something that's actually a sentence, and we'll get that merged.

**Review by sameo — APPROVED** [review]

**Pull request merged**